test-resources/bgp/expected_full_report.txt

```diff
@@ -48,17 +48,17 @@ Announcements from an authorized ASN, which are too specific (not allowed by max
 
 	Announcement: 10.0.0.0/24 => 64496
 
-		Disallowed by authorization(s):
+		Disallowed by configuration(s):
```

Comment:
Should this perhaps say "ROA" configuration as elsewhere you are explicit that this is not some other sort of configuration?

Reply:
ack fixed